src/Umbraco.Tests.Integration/Umbraco.Infrastructure/Packaging/PackageInstallationTest.cs

```diff
@@ -66,9 +72,17 @@ public void Can_Read_Compiled_Package_Warnings()
             using var fileStream = File.OpenRead(packageFile);
             CompiledPackage package = PackageInstallation.ReadPackage(XDocument.Load(fileStream));
             InstallWarnings preInstallWarnings = package.Warnings;
-            Assert.IsNotNull(preInstallWarnings);
-
-            // TODO: More Asserts
+            var dataType = package.DataTypes.First();
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual("Document Type Picker", package.Name);
+                Assert.AreEqual("3593d8e7-8b35-47b9-beda-5e830ca8c93c", dataType.LastAttribute.Value);
+                Assert.AreEqual("Document Type Picker", dataType.FirstAttribute.Value);
```

Comment:
```suggestion
                Assert.AreEqual("3593d8e7-8b35-47b9-beda-5e830ca8c93c", dataType.LastAttribute?.Value);
                Assert.AreEqual("Document Type Picker", dataType.FirstAttribute?.Value);
```

Possible null reference exception, with this change you would see something like "expected Document Type Picker, but got null", instead of a null reference exception, not a major thing, but it's a bit easier to understand than a null reference  exception 😄